raptorWeb/raptorWeb/settings.py

```diff
@@ -240,6 +242,11 @@
             'handlers': ['console', 'log_file'],
             'level': 'DEBUG',
             'propagate': False,
+        },
+        'raptorbot.botware': {
+            'handlers': ['console', 'log_file'],
+            'level': 'DEBUG',
```

Comment:
Configure proper logging for raptorbot actual bot code, currently it is using a primitive logger